src/app/example_configs/roles.yaml

```diff
@@ -0,0 +1,44 @@
+- name: "job_manager"
```

Comment:
The details of this file don't entirely matter right now.  It's just an example config that over time we'll build out to be whatever we think are reasonable default roles.